frontend/server/src/main/java/org/pytorch/serve/util/ConfigManager.java

```diff
@@ -1203,6 +1230,10 @@ public String getWorkflowStore() {
             return workflowStore;
         }
 
+        public String isTokenDisabled() {
```

Comment:
Why do we use a string here instead of boolean?

Reply:
Using this function to set prop and prop takes strings not booleans.